content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,109 @@
+[grafixeyehero](https://github.com/grafixeyehero).
+
+You can check the redesign in our [demo server](https://demo.jellyfin.org/stable)
+
+## Faster than the sound and cleaner than a surgery room
+
+As you already know, we're still a really young community that want to provide the best self-hosted media server out there. As with every release,
+we tried to clean up and optimize the old code that we inherited when we forked Emby's code. And that's what we have been doing since
+the day we released 10.4.3. A lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.
+
+Let's start with the frontend, as it's what you see everyday in the end. A lot of work has been done to simplify the code and ease the transition
+to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. For you, end users, this probably is all rubbish because there is nothing you feel that affects you right now. Plain wrong. Just see this video:
+
+<video autoplay muted loop preload="none" id="preview">
+    <source src="/images/10.5-release/speed.mp4" type="video/mp4">
+</video>
+
+Not only you're seeing the speed differences, you're seeing the new animations! Images will fade in every time you move to another page. You can disable that in settings if you want to come back the old behaviour though.
+
+The old-standing issue that  many of you had (by the way, introduced by the fellow writer of this post) caused by the images has been fixed in this version,
+although we want to keep improving in that matter, as we're still not completely happy with the current implementation. 
+
+That's pretty much what you'll notice at first sight for web anyway. A lot of not-that-relevant improvements, translations and bug fixes have been added. Consider that they're easter eggs, so happy finding! Ah, and we also added keyboard support for TVs and improved the compatibility with old browsers (just in case)!
+
+Okay, server time then! This time we focused heavily in migrating to .NET Core 3.0 (the framework we use for building the server's core).
+Although this seems boring, let's see some benchmarks from the hex decoder (used in server's tasks):
+
+```
+|                         Method |     Mean |    Error |   StdDev |   Gen 0 | Gen 1 | Gen 2 | Allocated |
+|------------------------------- |---------:|---------:|---------:|--------:|------:|------:|----------:|
+|                New hex decoder | 10.81 us | 0.079 us | 0.074 us |  0.1068 |     - |     - |     336 B |
+|                Old hex decoder | 23.57 us | 0.055 us | 0.051 us | 11.2305 |     - |     - |   35272 B |
+```
+*Shoutout to [Bond-009](https://github.com/Bond-009) for this!*
+
+That's only a tiny example of what we have been cooking and the groundwork preparation we've been doing for upcoming versions.
+However, there are some key improvements here, which are mostly focused in performance:
```

Comment:
However, the majority of improvements have been focused on performance and bug fixes as usual: